config/initializers/acts_as_taggable_on.rb

```diff
@@ -0,0 +1 @@
+ActsAsTaggableOn.force_lowercase = true
```

Comment:
🔥 do we have upper cased tags in the DB?

Reply:
yes we currently do 😬